opencti-platform/opencti-graphql/tests/03-streams/00-Raw/raw-test.js

```diff
@@ -42,11 +42,11 @@ describe('Raw streams tests', () => {
       expect(createEventsByTypes.file.length).toBe(4);
       expect(createEventsByTypes.campaign.length).toBe(5);
       expect(createEventsByTypes.incident.length).toBe(2);
-      expect(createEventsByTypes.report.length).toBe(31);
+      expect(createEventsByTypes.report.length).toBe(33);
```

Comment:
why +2 reports? I only see 1 new report created in taskManager-test, where is the other one?